src/widgets/MeterPanel.cpp

```diff
@@ -1897,13 +1897,6 @@ void MeterPanel::StartMonitoring()
    }
 }
 
-void MeterPanel::StopMonitoring(){
-   mMonitoring = false;
-   auto gAudioIO = AudioIO::Get();
-   if (gAudioIO->IsMonitoring()){
-      gAudioIO->StopStream();
-   } 
-}
```

Comment:
I think this deletion was a change for debugging purposes that you don’t want to merge.  The stopping level of monitoring with a click on the meter is disabled and not restored again.  Do I misunderstand something?

Reply:
I deleted this because it was not used and it is confusing. What gets called in response to either starting or stopping monitoring, is always MeterPanel::StartMonitoring.